Makefile

```diff
@@ -163,11 +164,11 @@ docs:
 	@echo "🥫 Generationg doc…"
 	${DOCKER_COMPOSE} run --rm --no-deps api ./build_mkdocs.sh
 
-checks: toml-check flake8 black-check mypy isort-check docs
+checks: toml-check flake8 black-check mypy isort-check
```

Comment:
why did you remove docs from checks ? Because they are built in another workflow ?